torch/lib/TH/generic/THTensor.h

```diff
@@ -12,6 +12,8 @@ typedef struct THTensor
     long *stride;
     int nDimension;
 
+    // Note: storage->size may be greater than the recorded size
```

Comment:
We can remove, if the lack of this invariant is obvious to everyone except me :) (But if this is the most important TH data structure, it should be well commented, right? :)